Game/marvin.cpp

```diff
@@ -130,6 +136,25 @@ bool Marvin::exec(Gothic& gothic, const std::string& v) {
         c->setToogleEnable(!c->isToogleEnabled());
       return true;
       }
+    case C_Insert: {
+      bool ret = false;
```

Comment:
`ret` can be removed: if world is null or something - you can just early-out by `return false` straight

Reply:
Its a habit from work where we have to be MISRA complient to a certain extend.
And one MISRA rule is that early exit is not allowed, which has its benefits for example you dont need to set breakpoints every where and you are certain that the code  before the last return is executed every time if you have cleanup code at the end of the function for example.

But no problem will change that.